.github/workflows/ubuntu-cve-monitor.yaml

```
@@ -155,56 +155,36 @@ jobs:
-          # Create staging notification for Ubuntu security fix trigger
-          SLACK_MESSAGE=$(cat <<EOF
-          🔐 *Staging Build Triggered - Ubuntu Security Fix*
-
-          📦 *Version:* \`$VERSION\`
-          🔄 *Trigger:* Automated Ubuntu 24.04 Security Fix (Scheduled)
-
-          ⚙️ Staging base images build has been triggered to address Ubuntu 24.04 security notices.
+          SLACK_MESSAGE="🔐 *Ubuntu 24.04 Security Refresh*
 
-          This is an *automated security refresh*, not a manual release.
+📦 Version \`$VERSION\` - Build started
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)